website/docs/r/lab_service_schedule.html.markdown

```diff
@@ -0,0 +1,110 @@
+---
+subcategory: "Lab Service"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_lab_service_schedule"
+description: |-
+  Manages a Lab Service Schedule.
+---
+
+# azurerm_lab_service_schedule
+
+Manages a Lab Service Schedule.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_lab_service_lab" "example" {
+  name                = "example-lab"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  title               = "Test Title"
+
+  security {
+    open_access_enabled = false
+  }
+
+  virtual_machine {
+    admin_user {
+      username = "testadmin"
+      password = "Password1234!"
+    }
+
+    image_reference {
+      offer     = "0001-com-ubuntu-server-focal"
+      publisher = "canonical"
+      sku       = "20_04-lts"
+      version   = "latest"
+    }
+
+    sku {
+      name     = "Classic_Fsv2_2_4GB_128_S_SSD"
+      capacity = 1
+    }
+  }
+}
+
+resource "azurerm_lab_service_schedule" "example" {
+  name         = "example-labschedule"
+  lab_id       = azurerm_lab_service_lab.example.id
+  stop_at      = "2022-11-28T00:00:00Z"
+  time_zone_id = "America/Los_Angeles"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Lab Service Schedule. Changing this forces a new resource to be created.
+
+* `lab_id` - (Required) The resource ID of the Lab Service Schedule. Changing this forces a new resource to be created.
+
+* `stop_at` - (Required) When Lab User Virtual Machines will be stopped in RFC-3339 format.
+
+* `time_zone_id` - (Required) The IANA Time Zone ID for the Schedule.
+
+* `notes` - (Optional) The notes for the Schedule.
+
+* `recurrence_pattern` - (Optional) A `recurrence_pattern` block as defined below.
+
+* `start_at` - (Optional) When Lab User Virtual Machines will be started in RFC-3339 format.
```

Comment:
since this is optional what does it default to? we should detail that

and to be explicit
```suggestion
* `start_time` - (Optional) When Lab User Virtual Machines will be started in RFC-3339 format.
```

Reply:
No default value for it.